### PR TITLE
Battlefield news

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: gamenews [hunk 1]
```diff
@@ -1,4 +1,4 @@
-2022-12-03 13-00-00:
-	Title: Playtest 20221203
+2022-12-23 23-00-00:
+	Title: Playtest 20221223
 	Author: Lukas Franke
```
Comment (abcdefg30) [inline gamenews:3]: You can change the name too if you want.

Comment (PunkPun) [reply gamenews:3]: If you insist

**Review by abcdefg30 — APPROVED** [review]
Preview:
grafik

It's a bit long so there is a scroll bar, but that is ok imo.

**Comment by PunkPun** [comment]
Since it's a recap that's taking all the length it's fine. I guess I could also make it longer on purpose by moving the recap line to a newline

**Comment by PunkPun** [comment]
How do you preview? For me either it gets overwritten by the old announcement or doesn't appear at all

**Comment by PunkPun** [comment]
Screenshot 2022-12-23 at 21 01 27

**Pull request merged**